test/tests/performance/pbs_jobperf.py

```diff
@@ -95,6 +95,19 @@ def wait_for_job_finish(self, jobiter=True):
         self.server.expect(JOB, {'job_state=F': total_jobs}, extend='x',
                            interval=20, trigger_sched_cycle=False)
 
+    def delete_jobs_per_user(self, users, num_users):
```

Comment:
Instead of calling this function why don't you call self.server.cleanup_jobs(runas='root')?

Reply:
This deletes jobs one by one for 10k jobs i took around 30 mins , qdel `qselect` would delete 100k by that time. 100k jobs might be a lot to delete at once so i am calling it per user.

Reply:
we should fix cleanup_jobs() because it makes no sense for it to take longer than what you are doing

Reply:
Sure. I think we should delete jobs in chunks when jobs are greater than 100 or 1000. I will log a ticket to improve it.